openssl-static/pom.xml

```diff
@@ -0,0 +1,214 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2016 The Netty Project
+  ~
+  ~ The Netty Project licenses this file to you under the Apache License,
+  ~ version 2.0 (the "License"); you may not use this file except in compliance
+  ~ with the License. You may obtain a copy of the License at:
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  ~ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  ~ License for the specific language governing permissions and limitations
+  ~ under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.netty</groupId>
+    <artifactId>netty-tcnative-parent</artifactId>
+    <version>1.1.33.Fork13-SNAPSHOT</version>
+  </parent>
+  <artifactId>netty-tcnative-openssl-static</artifactId>
+  <packaging>jar</packaging>
+
+  <name>Netty/TomcatNative [OpenSSL - Static]</name>
+  <description>
+    A Mavenized fork of Tomcat Native which incorporates various patches. This artifact is statically linked
+    to OpenSSL and Apache APR.
+  </description>
+
+  <properties>
+    <opensslBuildDir>${project.build.directory}/openssl-${opensslVersion}</opensslBuildDir>
+    <linkStatic>true</linkStatic>
+  </properties>
+
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <executions>
+          <!-- Download and extract the OpenSSL archive -->
+          <execution>
+            <id>get-openssl</id>
+            <phase>generate-sources</phase>
+            <goals>
+              <goal>run</goal>
+            </goals>
+            <configuration>
+              <target>
+                <ftp action="get"
+                     server="ftp.openssl.org"
+                     remotedir="source"
+                     userid="anonymous"
+                     password="anonymous"
+                     passive="yes"
+                     verbose="yes">
+                  <fileset dir="${project.build.directory}">
+                    <include name="**/openssl-${opensslVersion}.tar.gz"/>
+                  </fileset>
+                </ftp>
+                <gunzip src="${project.build.directory}/openssl-${opensslVersion}.tar.gz"
+                        dest="${project.build.directory}/" />
+                <untar src="${project.build.directory}/openssl-${opensslVersion}.tar"
+                       dest="${project.build.directory}/" />
+                <!-- The untar task does not preserve permissions. Make everything executable. -->
+                <chmod perm="ug+wx" type="file">
+                  <fileset dir="${opensslBuildDir}/" includes="**/*" />
```

Comment:
It was getting to be a game of wack-a-mole, unfortunately.  I had tried that initially, but it was a bit frustrating :/.